compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -551,14 +567,16 @@ LogicalResult setMatmulOpConfig(spirv::ResourceLimitsAttr limits,
   const int subgroupSize = limits.getSubgroupSize();
   const int maxBytes = limits.getMaxComputeSharedMemorySize();
 
-  auto pipeline =
+  // Force a 2 stage pipeline by default
```

Comment:
Hmm, this is not really forcing a 2 stage pipeline? It's just guarding against `0` AFAICT.

Reply:
I noticed that sometimes with tuning configs it would pass 0 but the default before this patch is depth 1 which leads to a 2 stage pipeline. This guard is just to make sure we default to that same 2 stage pipeline without multi-buffering, but I suppose the comment here is a bit confusing.

Reply:
I updated the comment here, let me know if it makes sense, but essentially I didn't want to change the default in case of interfering with tuning configs for LLVMGPU.